Makefile

```diff
@@ -79,7 +79,7 @@ lint-python:
 lint-html:
 	@yarn htmlhint "testCourse/**/question.html" "exampleCourse/**/question.html" "site"
 lint-markdown:
-	@yarn markdownlint "docs/**/*.md"
+	@yarn markdownlint --ignore node_modules --ignore exampleCourse --ignore testCourse --ignore "**/dist/**" "**/*.md"
```

Comment:
I was concerned at first that this would lint Markdown files in `.changesets`, which will intentionally violate some rules, e.g. that files have headings. However, it looks like dotfiles/dot-directories are ignored by default.